server/core/session_verification.py

```diff
@@ -33,10 +33,14 @@ def verify_session(self, packet, address):
         """
         Verifies a session
         """
-        if self.server.handle_until_value('auth:check_token', True, packet.username, packet.session_key):
-            self.server.connections[address]['uid'] = self.server.handle_until_value('auth:get_user_id', True, address)
-        else:
-            self.server.rnserver.close_connection(address)
+        pass
+        # TODO: uncomment when connection stuff is fixed
+        # if self.server.handle_until_value('auth:check_token', True, packet.username, packet.session_key):
+        #     uid = self.server.handle_until_return('auth:get_user_id', address)
```

Comment:
Ok, I found the problem with this. Should be `handle_until_return` not `handle_until_value`